default/scripting/ship_parts/General/SP_PLANET_DESTROYER.focs.txt

```diff
@@ -7,7 +7,23 @@ Part
     buildtime = 8
     tags = [ "CTRL_ALWAYS_BOMBARD" ]
     location = All
-    effectsgroups = [    
+    effectsgroups = [
+
+        // pdc gives speed malus increasing with refinement
+        EffectsGroup
```

Comment:
refinements should never give a penalty when researched

Reply:
Is that a set in stone rule? The idea is that the countdown decrease with each refinement is very powerful, and the increasing speed malus is supposed to even that out somewhat.

Reply:
> Is that a set in stone rule?

yes

Things players can pick to turn on / off can have penalties, not techs that can't be turned off.

Reply:
ok